.eslintrc.js

```diff
@@ -10,13 +10,10 @@ module.exports = {
     '@vue/typescript'
   ],
   rules: {
-    'no-console': 0,
+    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'max-len': ['warn', { code: 120 }],
-    'no-unused-vars': 'off',
-    'quotes': 'off',
-    'no-useless-return': 'off',
-    'camelcase': 'off',
-    'no-useless-escape': 'off'
+    'camelcase': 'off' // FUTURE: fix code and remove this
```

Comment:
I left this one because there were a ton of place to fix. Basically any props/methods that looked like `i_am_something` should be renamed `iAmSomething`, but this doesn't break anything.

I mean, it _could_ be fixed, but I'm not going to do it atm.